app/models/item.rb

```diff
@@ -28,7 +28,7 @@ def dollar_amount
   end
 
   def category_list
-    categories.map { |cat| cat.name }.join(", ")
+    categories.first
```

Comment:
I think we should ditch the many relationship on categories and just have them pick one.